x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_field.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ResultFieldValue } from '.';
+import { FieldType, Raw, Snippet } from './types';
+
+import './result_field.scss';
+
+interface Props {
+  field: string;
+  raw?: Raw;
+  snippet?: Snippet;
+  type?: FieldType;
+}
+
+export const ResultField: React.FC<Props> = ({ field, raw, snippet, type }) => {
+  return (
+    <div className="appSearchResultField" key={field}>
+      <div className="appSearchResultField__key eui-textTruncate">
+        <span>{field}</span>
+      </div>
+      <div className="appSearchResultField__separator" />
```

Comment:
This isn't an "unnecessary" comment but since we're already here in the ResultField component:
```suggestion
      <div className="appSearchResultField__separator" aria-hidden />
```

I noticed while doing screen-reader testing that Mac VO was reading out the `=>` symbol/content as "equals greater than" which actually really surprised me, since the last I had read up on it, CSS pseudo-elements `content` weren't parseable by screen readers - but I guess that's changed recently? In any case, I think adding an `aria-hidden` prop quickly takes care of that and brings the reading experience to the same as the header, e.g.

"engine" [ctrl option right arrow] "some engine"
"title" [ctrl option right arrow] "some title here"

etc.